testsuite/src/it/java/org/graalvm/tests/integration/DebugSymbolsTest.java

```diff
@@ -255,6 +255,9 @@ public void debugSymbolsQuarkusContainer(TestInfo testInfo) throws IOException,
             if (QUARKUS_VERSION.startsWith("2.4")) {
                 runCommand(getRunCommand("git", "apply", "quarkus_2.4.x.patch"),
                         Path.of(BASE_DIR, Apps.DEBUG_QUARKUS_BUILDER_IMAGE_VERTX.dir).toFile());
+            } else if (QUARKUS_VERSION.equals("999-SNAPSHOT")) {
```

Comment:
How about `... if (QUARKUS_VERSION.contains("SNAPSHOT")) { ...` ? Do we want to hard-code `999`?

Reply:
No opinion on my side, I just went that way since that's the default version used by Quarkus `main` for snapshot builds.

Reply:
Then just looking for `SNAPSHOT` should be good enough, I think. It's more future proof.